### DISCUSSION TITLE
Table: Nested table headers

### DISCUSSION THREAD

**Discussion by karlie93** [grafana/grafana#38145] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I hope that the option to merge and split cells can be added to the title of the table panel

**Why is this needed**:

In the process of using grafana for report display, merge and split cells are necessary and common requirements，
 hope grafana can support this option，Thanks!

![20200828142547](https://user-images.githubusercontent.com/54050952/91528894-e22a0100-e93a-11ea-919f-a8127018d9a8.png)

**Comment by torkelo** [comment]
sounds like you want nested / grouped table headers?

**Comment by karlie93** [comment]
yes, this feature is very common in our reporting needs.

**Reply by rouge-ware** [reply]
There needed In everyone's reporting, I wonder why it's not in the top1.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
